core/shared/src/main/scala/zio/internal/FiberContext.scala

```diff
@@ -1067,6 +1067,20 @@ private[zio] final class FiberContext[E, A](
     // For improved fairness, we resume in order of submission:
     observers.reverse.foreach(k => k(result))
   }
+
+  override def toString: String =
+    s"""
```

Comment:
Is the newline here going to be stripped off?

Reply:
Actually, no, I'll update!